src/xmldocvisitor.cpp

```diff
@@ -926,11 +926,11 @@ void XmlDocVisitor::operator()(const DocImage &img)
                 altValue, img.isInlineImage());
 
   // copy the image to the output dir
-  FileDef *fd;
+  FileDef *fd = 0;
   bool ambig;
   if (url.isEmpty() && (fd=findFileDef(Doxygen::imageNameLinkedMap,img.name(),ambig)))
   {
-    copyFile(fd->absFilePath(),Config_getString(XML_OUTPUT)+"/"+baseName);
+    if (fd) copyFile(fd->absFilePath(),Config_getString(XML_OUTPUT)+"/"+baseName);
```

Comment:
this extra `if (fd)` should not be needed, `fd` is already checked in the expression on the previous line.

Reply:
Indeed apparently `File *fd = 0` already removed the warning.